src/DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -1241,6 +1241,8 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
             return start_map>=90;
         case "PhoneCall":
             return start_map>100; //TODO: Last phone is in the building before the catacombs (Where Icarus calls)
+        case "FordSchick_PlayerDead":
+            return start_map >= 30;
```

Comment:
this was previously handled on line 1195, you just need to fix that line with the new name

probably others similar to that with the changes from _Dead to _PlayerDead